src/index.html

```diff
@@ -1,13 +1,420 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a
+              href="#menu"
+              class="icon icon--menu"
+            ></a>
+
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo.png"
+                alt="Miami Condo Kings">
+            </a>
+
+            <a class="icon icon--phone header__call" href="tel:+1 234 555-55-55"></a>
```

Comment:
phone icon is cropped
![image](https://user-images.githubusercontent.com/37161315/170643411-6c819294-5895-4d5a-85df-5497bf2c8488.png)